base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -1,29 +1,10 @@
 /*
- *  ReactOS Win32 Applications
- *  Copyright (C) 2007 ReactOS Team
- *
- *  This program is free software; you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation; either version 2 of the License, or
- *  (at your option) any later version.
- *
- *  This program is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License along
- *  with this program; if not, write to the Free Software Foundation, Inc.,
- *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
- */
-/*
- * PROJECT:         ReactOS Event Log Viewer
- * LICENSE:         GPL - See COPYING in the top level directory
- * FILE:            base/applications/mscutils/eventvwr/eventvwr.c
- * PURPOSE:         Event Log Viewer main file
- * PROGRAMMERS:     Marc Piulachs (marc.piulachs at codexchange [dot] net)
- *                  Eric Kohl
- *                  Hermes Belusca-Maito
+ * PROJECT:     ReactOS Event Log Viewer
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Event Log Viewer main file.
+ * COPYRIGHT:   Copyright 2007 Marc Piulachs (marc.piulachs at codexchange [dot] net)
+ *              Copyright 2008-2016 Eric Kohl
+ *              Copyright 2016-2022 Hermès Bélusca-Maïto
```

Comment:
I'd go without an ending dot in the 'PURPOSE' field. Also missing emails for last two authors :)
```suggestion
 * PURPOSE:     Event Log Viewer main file
 * COPYRIGHT:   Copyright 2007 Marc Piulachs <marc.piulachs@codexchange.net>
 *              Copyright 2008-2016 Eric Kohl <eric.kohl@reactos.org>
 *              Copyright 2016-2022 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
```

Reply:
What if I (or others) don't want to specify their emails? :oof:

Reply:
This is not an option, according to our coding style: https://reactos.org/wiki/Coding_Style

Reply:
Note that nothing says there whether email is mandatory or not. It just gives an example (in case there is a mail).

Reply:
Alright, but still I would prefer to stay uniform here. I had a similar discussion with Hervé, he didn't wanted to put an email because of too much spam coming to the inbox, but I don't see this as a solution of the problem.

Reply:
Yes this was exactly my intent. (Especially since I also don't consult reactos.org on a regular basis, but more like twice a year.)

Reply:
I do agree to binarymasters argumentation that it is better to omit the fullstop on the purpose line. Here and in the other files as well. But I won't block your PR just for that reason.